deps/rmutil/periodic.c

```diff
@@ -107,10 +107,16 @@ void RMUtilTimer_ForceInvoke(struct RMUtilTimer *t) {
 }
 
 int RMUtilTimer_Signal(struct RMUtilTimer *t) {
-  return pthread_cond_signal(&t->cond);
+  pthread_mutex_lock(&t->lock);
```

Comment:
Who calls `RMUtilTimer_Signal` can this locking effect the performance somehow?

Reply:
Currently, not in use. but i didn't want to change the API